ExampleApp Android/src/main/java/com/rootstrap/flowforms/example/SignUpFormActivity.kt

```diff
@@ -76,10 +79,15 @@ class SignUpFormActivity : AppCompatActivity() {
     }
 
     private fun onEmailStatusChange(status: FieldStatus) {
+        binding.emailAvailableText.visibility = View.GONE
+        binding.emailLoadingProgressBar.visibility = View.GONE
+        binding.emailInputLayout.error = null
```

Comment:
maybe `binding?.apply { ... }` can be used here?

Reply:
Yes! Nice catch